async_channel/util/channel_creator.py

```diff
@@ -48,7 +46,7 @@ async def create_channel_instance(
     set_chan_method: typing.Callable,
     is_synchronized: bool = False,
     **kwargs: dict
-) -> channels.Channel:
+) -> object:
```

Comment:
Do we have to ?

Reply:
cyclic imports otherwise, if it doesnt solve the bin issue, i will revert it